components/PostChooser/postchooser.js

```diff
@@ -19,7 +19,10 @@ export const PostChooser = ( props ) => {
 		onSelectPost,
 		modalLabel,
 		modalTitle,
-		postTypes = [ 'posts', 'pages' ], // Default post types to search.
+		postTypes = [
+			{ label: __( 'Posts' ), value: 'post' },
+			{ label: __( 'Pages' ), value: 'page' },
+		], // Default post types to search.
```

Comment:
if you like the idea of moving this into the Modal component, this could just be `postTypes,` to pass in whatever is passed to this component.